dev/bt_master_test.py

```diff
@@ -0,0 +1,14 @@
+import btcomms
+
+bt = btcomms.BlueCommMaster()
+bt.irq(lambda data: print('Received:', data))
+
+bt.scan()
+
+while not bt.comm._scan_complete:
```

Comment:
Test looks good, but could we incorporate this waiting within BlueCommMaster? Just so that applications themselves don't need to do it

Reply:
Moved to `BlueCommMaster` class.